pkg/mcs/exportspec.go

```diff
@@ -40,6 +41,16 @@ type GlobalProperties struct {
 	Ports []mcsv1a1.ServicePort `json:"ports"`
 }
 
+type CompatibleError struct {
+	OtherServiceCluster string
+	InterferingField    string
+}
+
+func (err *CompatibleError) Error() string {
```

Comment:
please
- drop the `Error` (it is an error by definition...) 
- start with a lower case letter
- remove quotes (or replace with tick (i.e., `'`) if you think them necessary 
- perhaps `field %s conflicts with value in cluster %s`?